packages/neos-ui-sagas/src/UI/ContentCanvas/index.js

```diff
@@ -44,7 +44,8 @@ export function * watchStopLoading({globalRegistry, store}) {
 export function * watchReload() {
     yield takeLatest(actionTypes.UI.ContentCanvas.RELOAD, function * (action) {
```

Comment:
idk but why is there no reducer of `actionTypes.UI.ContentCanvas.RELOAD` which updates the current iframe url (if in payload)

then we can use the same selector down in line 61 ... or we can put into the selector a modified temporal state ...